package/yast2-users.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Oct 11 12:10:21 UTC 2016 - hguo@suse.com
+
+- Fix the invocation of authentication configuration instance
+  (bsc#1000749).
+- 3.1.58
```

Comment:
Please renumber this to 3.1.57.1, because 3.1.58 is in 4be5ae3d872776304ce9a9cf2da65d551a54c9cb with different contents already.